### PR TITLE
[DOCS] Adds size and from parameters to data frame APIs

### PR DESCRIPTION
**Pull request by lcawl**
This PR adds "from" and "size" parameters to the "get data frame transforms" API and "get data frame transform statistics" API.

The descriptions are copied from https://www.elastic.co/guide/en/elasticsearch/client/java-rest/master/java-rest-high-dataframe-get-data-frame-transform.html

### CODE REVIEW

File: docs/reference/data-frames/apis/get-transform-stats.asciidoc [hunk 1]
```diff
@@ -54,6 +60,16 @@ see {stack-ov}/security-privileges.html[Security privileges] and
 
 ==== Examples
 
+The following example skips for the first five {dataframe-transforms} and
+gets usage information for a maximum of ten results: 
+
+[source,js]
+--------------------------------------------------
+GET _data_frame/transforms/_stats?from=5;size=10
```
Comment (dimitris-athanasiou) [inline docs/reference/data-frames/apis/get-transform-stats.asciidoc]: Should the `;` between from and size be a `&` here?

**Review by dimitris-athanasiou — COMMENTED** [review]
Looks good apart from a minor comment I wasn't sure about.

File: docs/reference/data-frames/apis/get-transform-stats.asciidoc (hunk 1 above)
Comment (lcawl) [reply docs/reference/data-frames/apis/get-transform-stats.asciidoc]: It worked in Kibana, but I'll test again

**Review by benwtrent — COMMENTED** [review]
We should also state somewhere that the return order is deterministic and is the natural ascending sorting of each transform's ID, i.e. `a` before `aa` before `ab`

**Comment by lcawl** [comment]
> We should also state somewhere that the return order is deterministic and is the natural ascending sorting of each transform's ID, i.e. `a` before `aa` before `ab`

@benwtrent Thanks for the suggestion. I've added a phrase to the results section in each API.

**Review by dimitris-athanasiou — APPROVED** [review]
LGTM

**Review by benwtrent — APPROVED** [review]

**Pull request merged**